app/models/stack.rb

```diff
@@ -68,16 +68,20 @@ def async_refresh_deployed_revision
   def update_deployed_revision(sha)
     return if deploying?
 
+    last_deploy = deploys_and_rollbacks.completed.last
```

Comment:
Good question. Seemed logical to add it, but I'm having a hard time to justify it.

And actually it made me figure out that `aborting` should be considered an active state, and `aborted` a `completed` one.

Also since we bail out if there is an active deploy. The `completed` is no long needed.

Fixing this.